CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing guidelines for `blade`
+
+## Development setup
+
+These steps should get you up and started for local development setup. Please ensure you've NodeJS and Yarn installed on your machine before proceeding.
+
+### Prerequisites
+
+- `cd` into `packages/blade`
+
+- Install dependencies:
+
+  ```sh
+  yarn
+  ```
+
+### Setting up web
+
+- Run storybook for web:
+
+  ```sh
+  yarn react
+  ```
+
+- That's it!
+
+### Setting up native
+
+- Get a tea, coffee or your favorite drink, you'll need it 😸
+
+- Install [`brew`](https://brew.sh/) if you don't have it installed
+
+- Install watchman:
+
+  ```sh
+  brew install watchman
+  ```
+
+- Install [XCode](https://reactnative.dev/docs/environment-setup#xcode).
+
+  > **Note:**
+  >
+  > Sometimes it can take a very long time for XCode to install. Check [here](https://apple.stackexchange.com/questions/427640/mac-app-store-xcode-download-stuck-at-installing) for troubleshooting.
```

Comment:
```suggestion
  > Sometimes it can take a very long time for Xcode to install. Check [here](https://apple.stackexchange.com/questions/427640/mac-app-store-xcode-download-stuck-at-installing) for troubleshooting.
```